pages/apply/[resident]/index.tsx

```diff
@@ -43,6 +42,13 @@ const ApplicationStep = (): JSX.Element => {
     currentResident === mainResident
   );
 
+  if (
+    currentResident !== mainResident &&
+    getAgeInYears(currentResident) >= 18
```

Comment:
Think it would be better to pass this is as extra param to `getApplicationSectionsForResident` and do any required changes there so that it's all in one place.